corehq/apps/ota/views.py

```diff
@@ -361,4 +361,9 @@ def form_valid(self):
 @login_or_digest_or_basic_or_apikey()
 @require_GET
 def heartbeat(request, domain, id):
-    return JsonResponse({})
+    # mobile needs this. This needs to be revisited to actually work dynamically (Sravan June 7, 17)
+    return JsonResponse({
+        "app_id": "value_copied_from_app_id_url_param",
```

Comment:
You can expect that mobile will always include a url param in this request with key `app_id`